public/index.html

```diff
@@ -6,7 +6,6 @@
     <meta name="viewport" content="width=device-width,initial-scale=1.0">
     <title>BC Business Registry</title>
     <link rel="icon" href="<%= BASE_URL %>favicon.png">
-    <link rel="preload" href="<%= BASE_URL %>fonts/materialdesignicons-webfont.927457ed.woff2" as="font" type="font/woff2" crossorigin>
```

Comment:
This is a good idea for a real font (ie, to display the initial page in the correct font ASAP) but this is an icon font which we don't need right away. So removing this should speed up the load a bit.